make/autoconf/flags-cflags.m4

```diff
@@ -441,7 +445,9 @@ AC_DEFUN([FLAGS_SETUP_CFLAGS_HELPER],
       -I${TOPDIR}/src/java.base/share/native/libjava \
       -I${TOPDIR}/src/java.base/$OPENJDK_TARGET_OS_TYPE/native/libjava \
       -I${TOPDIR}/src/hotspot/share/include \
-      -I${TOPDIR}/src/hotspot/os/${HOTSPOT_TARGET_OS_TYPE}/include"
+      -I${TOPDIR}/src/hotspot/os/${HOTSPOT_TARGET_OS_TYPE}/include \
+      -I${OUTPUTDIR}/vm/runtime \
+      -I${OUTPUTDIR}/vm/runtime/omr"
```

Comment:
These paths are incorrect for UMA builds, and there isn't a common prefix that works for both.
Suggest changing `custom-spec.gmk.in` to define `OPENJ9_JDK_INCLUDES` with `OPENJ9_VM_BUILD_DIR`:
```
ifeq (true,$(OPENJ9_ENABLE_CMAKE))
  OPENJ9_VM_BUILD_DIR = $(OUTPUTDIR)/vm/runtime
  OPENJ9_JDK_INCLUDES = \
	-I$(OPENJ9_VM_BUILD_DIR) \
	-I$(OPENJ9_VM_BUILD_DIR)/omr
else
  OPENJ9_VM_BUILD_DIR = $(OUTPUTDIR)/vm
  OPENJ9_JDK_INCLUDES = \
	-I$(OPENJ9_VM_BUILD_DIR)/include \
	-I$(OPENJ9_VM_BUILD_DIR)/omr/include_core
endif
```
Then this can use that macro:
```
    \$(OPENJ9_JDK_INCLUDES)
```

Reply:
Actually, another arguably better approach, would be to copy `j9cfg.h` and `omrcfg.h` to `$(SUPPORT_OUTPUTDIR)/modules_include/java.base` which is already on the include path.

Reply:
Copied `j9cfg.h` & `omrcfg.h` into `$(SUPPORT_OUTPUTDIR)/modules_include/java.base`.